packages/fluentui/react-northstar/src/components/Form/FormFieldCustom.tsx

```diff
@@ -0,0 +1,182 @@
+import { Accessibility, FormFieldBehaviorProps, formFieldBehavior } from '@fluentui/accessibility';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import {
+  childrenExist,
+  UIComponentProps,
+  ChildrenComponentProps,
+  commonPropTypes,
+  getOrGenerateIdFromShorthand,
+  createShorthand,
+} from '../../utils';
+import { ShorthandValue, withSafeTypeForAs, ProviderContextPrepared } from '../../types';
+
+import Box from '../Box/Box';
+import {
+  getElementType,
+  useTelemetry,
+  useStyles,
+  useAccessibility,
+  compose,
+} from '@fluentui/react-bindings';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+import FormLabel, { FormLabelProps } from './FormLabel';
+import FormMessage, { FormMessageProps } from './FormMessage';
+
+export interface FormFieldCustomProps extends UIComponentProps, ChildrenComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   */
+  accessibility?: Accessibility<FormFieldBehaviorProps>;
+
+  /** A field can have its label next to instead of above it. */
+  inline?: boolean;
+
+  /** The HTML input id. This will be set on the control element and will be use for linking it with the label for correct accessibility. */
+  id?: string;
+
+  /** A label for the form field. */
+  label?: ShorthandValue<FormLabelProps>;
+
+  /** Text message that will be displayed below the control (can be used for error, warning, success messages). */
+  message?: ShorthandValue<FormMessageProps>;
+
+  /** Message to be shown when input has error */
+  errorMessage?: ShorthandValue<FormMessageProps>;
+}
+
+export const formFieldClassName = 'ui-form__field';
```

Comment:
Should it be a different classname, not to overlap with existing `FormField` component?

Reply:
Changed.